docs/blog/2025-january-update.rst

```diff
@@ -0,0 +1,58 @@
+.. post:: Jan 7, 2025
+   :tags: stats, year in review, newsletter
+   :author: Eric Holscher
+
+Write the Docs Community Update - January 2025
+===============================================
+
+Hey folks, Eric Holscher here, one of the co-founders of Write the Docs.
+I'm here with our traditional January community update while the newsletter team takes the month off.
+
+2025 is going be mostly a continuation of our efforts from last year.
+We don't have any major new initiatives planned,
+but we're continuing to refine our approach to conferences and building community.
+
+2024 Salary Survey ends soon
+----------------------------
+
+The `2024 Salary Survey <https://salary-survey.writethedocs.org/>`_ is still open until January 15.
+We've had less responses than last year so far, 
+so please take a moment to fill it out or share it with a friend if you haven't already.
+We plan to release the results in Q2, in time for the Portland conference.
+
+Our 2025 conferences
+--------------------
+
+It was wonderful to see everyone at our 2024 Portland, Atlantic, and Australia conferences.
+Our 2025 conferences will look similar to last year:
+
+* `Portland <https://www.writethedocs.org/conf/portland/2025/>`_ will be in-person on **May 4-6**.
+* **Atlantic** will be a virtual event, spanning the East Coast and European timezones. We might push it a bit later in the year to avoid the many conferences that happen in mid Septermber, but haven't firmed up dates.
+* Australia just happened a month ago, and we haven't started planning for 2025 yet.
+
+We are currently accepting `talk proposals <https://www.writethedocs.org/conf/portland/2025/cfp/>`_ and selling tickets for our Portland conference.
+**Submit your talk proposals by January 21**.
+Ticket sales will remain open until the week before the conference. 
+
+We hope that you'll work with us to make all of our events unique again this year by submitting talks and showing up in-person or virtually.
+
+Community board updates
+-----------------------
+
+The `Community Board <https://www.writethedocs.org/team/#community-board>`_ has been instrumental in guiding our strategic direction and supporting our community teams.
+Here are some highlights from 2024:
+
+* **Jobs and hiring support:** To better align with changing trends in the documentarian jobs market, we removed the job board and launched a new job sponsorship offering. We are also reviewing job-related Slack spaces to ensure they serve the community effectively.
+* **Meetups:** The meetups team is auditing active meetup.com accounts and exploring alternatives to meetup.com. They are also updating guidelines for WTD sponsorship of meetup.com accounts.
+* **Slack moderation:** Our moderation team continues to fight spam and maintain a welcoming environment in our Slack community. We are exploring ways to improve the onboarding experience for new members, but overall the community is in a good place.
```

Comment:
```suggestion
* **Slack moderation:** Our moderation team continues to maintain a welcoming environment and fight spam in our Slack community. We are exploring ways to improve the onboarding experience for new members, but overall the community is in a good place.
```
Very nitpicky, but reading "Our moderation team continues to fight" just felt slightly suboptimal to me.